testng-core/src/main/java/org/testng/reporters/AbstractXmlReporter.java

```diff
@@ -200,7 +200,8 @@ protected final Count computeCountForSuite(ISuite s) {
       }
       skipped += skippedPerTest;
       retried += retriedPerTest;
-      ignored += testContext.getExcludedMethods().stream().filter(ITestNGMethod::isTest).count();
+      ignored +=
+          (int) testContext.getExcludedMethods().stream().filter(ITestNGMethod::isTest).count();
```

Comment:
Why do you cast?

Reply:
Followed the IDE's recommendations and also the learnings from https://stackoverflow.com/a/51632153